packages/@react-spectrum/calendar/stories/RangeCalendar.stories.tsx

```diff
@@ -74,9 +74,8 @@ export default {
       control: 'select',
       options: [null, 'single', 'visible']
     },
-    validationState: {
-      control: 'select',
-      options: [null, 'valid', 'invalid']
+    isInvalid: {
```

Comment:
these (and a few others in this PR) are RSP components, they should still be validationState? or is the idea that these are components that don't have a valid validation state, only invalid?

Reply:
from pr description:

> Spectrum retains validationState only for input field like components, but moves to isInvalid for others such as Checkbox and Picker